lists/relay/adblock/adaway

```diff
@@ -0,0 +1,10938 @@
+
+  localhost
+::1  localhost
```

Comment:
Seems weird to have localhost in our blocklist?

Reply:
See latest PR. Have fixed.